build_scripts/windows/scripts/az.ps1

```diff
@@ -0,0 +1,2 @@
+$env:AZ_INSTALLER="MSI"
+& "$PSScriptRoot\..\python.exe" -IBm azure.cli $args
```

Comment:
This doesn't work correctly with WinPS 5.1, right?  Unfortunately the case of `az '"test"'` by the time the `$args` is processed, you would only get `test` on any PS older than 7.3, and there's no way to know what the original intent.  So perhaps the best thing is to iterate through `$args` and if `|` or `&` is found, throw an error that it would only work with 7.3+?

Reply:
Tested in Windows PowerShell 5.1, and indeed double quotes are lost, but luckily `|` or `&` isn't causing any problems to us. :)

```
> az2 --debug "a|b" "a&b" '"ab"'
cli.knack.cli: Command arguments: ['--debug', 'a|b', 'a&b', 'ab']
```